anki_addon/sonaveeb.py

```diff
@@ -100,10 +103,11 @@ def _parse_lexeme_definition(self, definition_row):
             return definition, level
 
         # Extract language level if present
-        for meta_span in definition_row.find_all(class_='additional-meta'):
-            if meta_span.get('title') == 'Keeleoskustase':
-                level = meta_span.string.strip()
-                break
+        if level := definition_row.find(class_='additional-meta', title='Keeleoskustase'):
+            level = level.string.strip()
+            # TODO: Is this check necessary? Are there words with invalid level value?
+            if level not in VALID_LANGUAGE_LEVELS:
+                level = None
```

Comment:
Is checking against `VALID_LANGUAGE_LEVELS` necessary? Have you seen words with invalid level values?

Reply:
It's not necessary; I've never seen other values here.